das-jdbc/src/main/java/org/wso2/das/jdbcdriver/dasInterface/DASServiceConnector.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.wso2.das.jdbcdriver.dasInterface;
 
+import javax.net.ssl.*;
```

Comment:
Avoid use imports with *. Always use the required imports only to have clean and unambiguous code.